tests/integration/targets/latest/tasks/device_interface.yml

```diff
@@ -288,3 +288,25 @@
     that:
       - test_eleven is failed
       - test_eleven['msg'] == "Must set update_vc_child to True to allow child device interface modification"
+
+
+- name: "12 - Interface with status"
+  networktocode.nautobot.device_interface:
+    url: "{{ nautobot_url }}"
+    token: "{{ nautobot_token }}"
+    device: test100
+    name: GigabitEthernet4
+    status: active
+    type: "1000Base-T (1GE)"
+  register: test_twelve
+
+- name: "12 - ASSERT"
+  assert:
+    that:
+      - test_twelve is changed
+      - test_twelve['msg'] == "interface GigabitEthernet4 created"
+      - test_twelve['diff']['before']['state'] == 'absent'
+      - test_twelve['diff']['after']['state'] == 'present'
+      - test_twelve['interface']['name'] == "GigabitEthernet4"
+      - test_twelve['interface']['status'] == "active"
+      - test_twelve['interface']['device'] == test100['key']
```

Comment:
Please implement the following:
- Add the `api_version` option to this test
- Choose a different status than the default of active to verify it actually sets it (like `planned`)
- Add a `when` conditions to only run these steps when Nautobot is version 1.4+. ([Example here](https://github.com/nautobot/nautobot-ansible/blob/develop/tests/integration/targets/latest/tasks/location.yml#L147))